test/runnable/structlit.d

```diff
@@ -1,8 +1,10 @@
 /*
+REQUIRED_ARGS: -preview=rvaluerefparam
```

Comment:
Isn't it questionable to change existing tests for a preview feature? This risks breaking the non-preview version.

Can the changes be versioned and the preview flag be part of the permutations?

Reply:
I don't think so, the only test changed that is not exclusively testing that rvals can't be passed to `ref` is the test for 8976, which is a trivial null pointer ICE.

They probably could but I don't think its worth it.

Reply:
> They probably could but I don't think its worth it.

I think it's worth it.  What about adding a `__trait(getCompilerFlags)`?  I think it'd probably have a number of other use cases.